src/Assetic/Extension/Twig/TwigFormulaLoader.php

```diff
@@ -34,7 +34,7 @@ public function __construct(\Twig_Environment $twig, LoggerInterface $logger = n
     public function load(ResourceInterface $resource)
     {
         try {
-            $tokens = $this->twig->tokenize($resource->getContent(), (string) $resource);
+            $tokens = $this->twig->tokenize(new \Twig_Source($resource->getContent(), (string) $resource), (string) $resource);
```

Comment:
I meant the second parameter to `tokenize` itself, not of `\Twig_Source`.

See https://github.com/twigphp/Twig/blob/v1.27.0/lib/Twig/Environment.php#L601, `$name` is superfluous. (Also to be forward-compatible, because in Twig master, the second parameter is gone: https://github.com/twigphp/Twig/blob/master/lib/Twig/Environment.php#L442)

So this line should be:
```php
            $tokens = $this->twig->tokenize(new \Twig_Source($resource->getContent(), (string) $resource));
```